### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Adarsh-jaiss**
Omitted and some words so that reader could get a better reading experience 🙂

### CODE REVIEW

File: docs/key-features/discussions.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 3
 
 ## What is a discussion?
 
-Discussions are a great way to get engaged with the community around various topics! This feature is here to help add extra context to every post. 
+Discussions are the great way to get engaged with the community around various topics! This feature is here to help by adding extra context to every post.
```
Comment (FrancescoXX) [inline docs/key-features/discussions.md:9]: the original one was correct

File: docs/key-features/discussions.md [hunk 2]
```diff
@@ -45,19 +45,19 @@ Including:
...
+All the details about this article including title, publish date, reading time, tags, etc.
 
 ### Table of contents
 
 daily.dev uses the headings in each article to build an automatic table of contents. That's a cool way to get a snapshot of the article and understand what it's about. 
 
 ### Suggested similar posts
 
-Based on the article's data, daily.dev will also recommend further reading recommendations. 
+Based on the article's data, daily.dev also recommends further reading recommendations.
```
Comment (FrancescoXX) [inline docs/key-features/discussions.md:60]: Original was correct

File: docs/key-features/discussions.md (hunk 1 above)
Comment (Adarsh-jaiss) [reply docs/key-features/discussions.md:9]: Should I close the pr? 🙄

Comment (FrancescoXX) [reply docs/key-features/discussions.md:9]: yes please

**Comment by GrahamTheDev** [comment]
Thanks for the PR, as per conversation with Francesco I am closing this PR.

**Pull request closed without merging**